openshift/openshift/src/main/java/org/arquillian/cube/openshift/impl/client/CubeOpenShiftConfiguration.java

```diff
@@ -140,47 +135,4 @@ public static CubeOpenShiftConfiguration fromMap(Configuration c, Map<String, St
             }
         }
     }
-
-    public static CubeOpenShiftConfiguration fromMap(Map<String, String> map) {
```

Comment:
There were two fromMap() methods.

- one with map arguments
- one with map and configuration arguments.

Since the first was no longer used (for a while) I removed that.